src/io.c

```diff
@@ -195,14 +195,44 @@ Char GET_NEXT_CHAR(void)
     }
     else
         STATE(In)++;
+
+    // handle line continuation, i.e., backslash followed by new line or CRLF;
+    // and also the end of a line in general
+    while (*STATE(In) == '\\' || *STATE(In) == 0) {
```

Comment:
Out of interest, why do we handle null characters like backslashes?

Reply:
We don't, we just have to keep reading for both: for null chars, we have to read more data via `GetLine()`, then start over; and for backslash, we have to see if next comes a lineend, and if so, skip the backslash and the lineend. And in theory, either of these can happen after the other, hence the loop.

Reply:
Does that make sense? Would a better comment help? Suggestions welcome!

Reply:
Reading this again, I didn't realise that the second part of 201 and 202 are the same case. Maybe change 202 to `if(*STATE(In) == 0)`, which would make it more obvious that is matches the second condition in 201?

Reply:
will do, and will try to write some comments, too

Reply:
Done